src/Workspaces/VisualBasic/Portable/Formatting/Engine/Trivia/TriviaDataFactory.Analyzer.vb

```diff
@@ -71,19 +72,25 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Formatting
                 result.Tab = 0
             End Sub
 
-            Private Shared Sub AnalyzeLineBreak(trivia As SyntaxTrivia, ByRef result As AnalysisResult)
-                ' if there was any space before line break, then we have trailing spaces
-                If result.Space > 0 OrElse result.Tab > 0 Then
-                    result.HasTrailingSpace = True
+            Private Shared Sub AnalyzeLineBreak(previousTrivia As SyntaxTrivia, trivia As SyntaxTrivia, ByRef result As AnalysisResult)
+                ' if there was any space immediately before line break, then we have trailing spaces
+                Dim text = previousTrivia.ToString()
```

Comment:
this value isn't used unless the previous trivia was whitepsace.  please move ito the 'if' block.

Reply:
Removed it completely as its not needed all I need to know is if the trivia width > 0.